client/src/components/Slider/Slider.css

```diff
@@ -0,0 +1,38 @@
+.slider__cards {
+        display: flex;
+}
+
+.PersonCard{
+        width: 100%;
+        margin: 0 2%;
+}
+/* SETS THE SIZE OF THE CARDS */
+
+.slider {
+        display: flex;
+}
+
+/* FOCUS AREA OF CARDS BEING DISPLAYED */
+
+.arrow__prev {
```

Comment:
You can simplify this allot by adding an `.arrow` class that contains most of these styles